docs/databags.md

```diff
@@ -93,7 +93,10 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `['bag']` - A string that points to an externalized Apps Hash in which all keys (except this one) are valid. 
 * `[app]` - The name of an app to manage (disk name)
 * `[app]['remove']` - If true, remove this app instead of creating / managing  (default - false)
-* `[app]['local']` - If true, manage files in the local directories instead of the "default" (default-false)
+* `[app]['local']` - If true, manage files in the local directories instead of the "default" (default - false, but forced true if download-url is specified)
```

Comment:
I think maybe I get it now.  When I was originally reading I was interpreting the attribute as having some effect on how the contents of the tarball itself would be treated.  It made me think as though the contents of the tarball are going to be treated as "local" even though typically what you'd put in a tarball would be "default".

I suspect this particular property applies the files described in [app]['files'] and whether those are treated as "local" or "default".  Naturally if you are installing a tarball and additionally defining [app]['files'] then what's in the tarball should be "default" and what is in [app]['files'] should be considered "local".

Maybe the change I can propose is to say the following:

If true, manage [app]['files'] defined files in the "local" directories instead of "default"...

The key difference there is I replaced the word "files" with "[app]['files']" to clarify which files exactly are being referred to.